### PR TITLE
Fix crash when beehive is broken by fake player

### PR DESCRIPTION
**Pull request by SquidDev**
When a beehive is broken, every nearby bee targets a random player. However, if there are no nearby players, the game crashes.

This should not occur under normal (vanilla) conditions. However, if a beehive is broken by a fake player there are no players in range, and so we see a crash.

### CODE REVIEW

**Review by apple502j — APPROVED** [review]
I actually have (unsuccessfully) attempted to crash the game in vanilla; yeah, this seems good.

**Review by Technici4n — APPROVED** [review]

File: fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/BeehiveBlockMixin.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+import net.minecraft.block.BeehiveBlock;
+import net.minecraft.entity.passive.BeeEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+@Mixin(BeehiveBlock.class)
+public class BeehiveBlockMixin {
```
Comment (Juuxel) [inline fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/BeehiveBlockMixin.java]:
```suggestion
class BeehiveBlockMixin {
```
Should be package-private according to CONTRIBUTING.md

**Review by Juuxel — APPROVED** [review]

File: fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/BeehiveBlockMixin.java (hunk 1 above)
Comment (SquidDev) [reply fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/BeehiveBlockMixin.java]: :+1: Someone may want to do a cleanup pass in the rest of this module - I believe the other mixins are also public there.

Comment (Juuxel) [reply fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/BeehiveBlockMixin.java]: Yeah, I believe old mixins (and most other code) weren't adjusted when the current contributing guidelines were added.

**Review by modmuss50 — APPROVED** [review]
Thanks for adding a test 👍

**Comment by SquidDev** [comment]
Thank you!

**Pull request merged**